ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -52,7 +52,7 @@ final case class Config[Extra](
     enableMutableContractStateCache: Boolean,
     enableInMemoryFanOutForLedgerApi: Boolean,
     extra: Extra,
-    enableSelfServiceErrorCodes: Boolean,
+    useLegacyErrorCodes: Boolean,
```

Comment:
Why rename this config parameter? We can keep it as previously and simply have the default changed + the CLI argument. You'll have a much smaller changeset.

Reply:
Changed it to have matching name between the config class and CLI param. Followed your advice though for less impact 👍